### ISSUE TITLE
Blogpost & Slackmassage about Mainnet Shutdown (pair programming)

### ISSUE THREAD

**Issue by simonweniger** [leapdao/meta#212]
# Bounty
Write a blogpost and a slackmassage about the mainnet shutdown

## Scope
- blogpost
- slackmassage

## Deliverables
- blogpost
- slackmassage

## Gain for the project
be transparent

## Roles
bounty gardener: @simonweniger  / 10%
bounty worker: @simonweniger @zschavi  / 70%
bounty reviewer: name / 20%

**Comment by zschavi** [comment]
# LeapDAO Mainnet Shutdown

### - Story of LeapDAO & Mainnet Start
In August 2017 Vitalik Buterin and Joseph Poon proposed **"Plasma"**, a scaling solution that enables Ethereum to reach a higher transaction output. When the Ethereum mainnet is busy, the execution speed for transactions is limited.

Plasma is here to bundle up transactions, and submit them to the mainnet in the form of periods, rather than blocks. Plasma enables fast transactions at a fraction of the cost.

The Ethereum Foundation awarded LeapDAO a grant to design and launch a working plasma chain, in order to tool up Ethereum with scalability.

Next to it, LeapDAO grew as a fully distributed team. Organized and equipped with holacratic principles, the team managed to realize a fully working plasma chain in 2 years.

### - Restructuring LeapDAO
After successfully implementing Plasma on Layer2, the team recognized a trend towards new and exciting technologies and decided to move one step forward.

Let's focus on Dapp development and provide the infrastructure for all Dapp projects willing to work on Ethereum. Next to Plasma there are other second layer solutions which offer a wide range of solutions already. 

The new approach is finding the right solution for the right project.

### - Research in Rollups
LeapDAO made their homework and has done extensive research in optimistic roll-ups & zK roll-ups. After getting familiar with the "new kids on the block", the team checked how to combine roll-ups with plasma knowledge and create a new powerful dark matter to scale Ethereum even further!
    
During the research it became obvious that every protocol is special and needs a custom solution in order to gain traction. Therefore the plasma chain will still be an interesting approach for specific scalable zero-knowledge proofs.

### - Improved development, only on the testnet
At the moment the leap mainnet is under construction. We concentrate our efforts on the functionality and reliability of the testnet.

The testnet is ready to get some traffic, and all products can be tested, before getting launched on main. It will all fall into place, once the brain/code work is done.

Soon, there will be a return of the mainnet with improved features and a variety of new updates. Maintaining a mainnet demands various resources: effort, money & manpower. All of these are rare and hard to obtain... 
    
### - Following our purpose and driving for mass adaption
Whatever is the way we go and which technologies we adopt, the team is motivated to work towards our common purpose:

"Help collectives of people to collaborate on things they care about and achieve sovereignty in a non-violent and fulfilling way."

The aim is to preserve our decentralized & fully-distributed team, get some engines running and help the crypto/ethereum ecosystem achieving mass-adoption.

**Comment by troggy** [comment]
Great writeup @zschavi 👏 

Couple of suggestions how to make "shutdown" to sound even less dangerous, if you don't mind.

What's worth mentioning:
- shutdown is scheduled on XX (put the date here). Before the date the chain is working normally.
- no fund loss is expected. We advise everyone to withdraw/exit their funds from Leap Mainnet before the shutdown date using normal plasma-procedure (https://mainnet.leapdao.org). For all the remaining funds, we plan to get custody of them and distribute them back on the Ethereum Mainnet (see below).

Write the **next steps** we are going to take. Namely:
  - we are submitting governance proposals (here goes link to blog article about Minimum Viable Governance) to upgrade Leap Plasma contracts to be able to recover unclaimed funds to distributed them back to their respective owners on the root chain. These proposals will also disallow further deposits.
   - once the contract upgrades are finalized, we do the actual shutdown by stopping all the Leap Mainnet validator nodes and all the related infrastructure.
   - we withdraw the funds from upgraded Plasma Contracts and distribute them on Ethereum Mainnet to their respective owners

Probably worth mention *Call to action* explicitly:
- if you have any funds on Leap Mainnet, we advise to withdraw/exit them using normal plasma-procedure (https://mainnet.leapdao.org)

**Comment by zschavi** [comment]
@troggy - thanks for the feedback and good add-on information 🔨 

What about this:

### - Hard facts & The next steps

The official mainnet shutdown is expected for the 1st of March 2020. Until then, the Leap Plasma chain is in operation and working. In full [holacracy fashion](https://leapdao.org/blog/Minimal-Viable-Governance), a governance proposal will be submitted, to enable the next proceedings.

Leap Plasma contracts will be upgraded and unclaimed funds are recovered, in order to distribute them to their respective owners via root-chain. Further deposits are not allowed.

Once the contract upgrades are finalized, the actual shutdown is done by stopping all the Leap Mainnet validator nodes and all the related infrastructure. Funds are withdrawn & returned.

--> If you have any funds on Leap Mainnet, please make sure to withdraw them. !! <--
As our mainnet pause is starting on the 01.03.2020, we advise you to take action before that date.

You can do that very easy over here: https://mainnet.leapdao.org

**Comment by zschavi** [comment]
final edit:

# LeapDAO Mainnet Shutdown

### - Plasma, LeapDAO & Mainnet Start
In August 2017 Vitalik Buterin and Joseph Poon proposed "Plasma", a scaling solution that enables Ethereum to reach a higher transaction output. When the Ethereum mainnet is busy, the execution speed for transactions is limited.

Plasma is here to bundle up transactions, and submit them to the mainnet in the form of periods, rather than blocks. Plasma enables fast transactions at a fraction of the cost.

The Ethereum Foundation awarded LeapDAO a grant to design and launch a working plasma chain, in order to tool up Ethereum with scalability.

Next to it, LeapDAO grew as a fully distributed team. Organised and equipped with holacratic principles, the team managed to realise a fully working plasma chain in 2 years.

### - Restructuring LeapDAO
After successfully implementing Plasma on Layer2, the team recognised a trend towards new and exciting technologies and decided to move one step forward.

Let's focus on DApp development and provide the infrastructure for all DApp projects willing to work on Ethereum. Next to Plasma there are other second layer solutions which offer a wide range of solutions already. 

The new approach is finding the right solution for the right project.

### - Research in Rollups
LeapDAO made their homework and has done extensive research in optimistic roll-ups & zK roll-ups. After getting familiar with the "new kids on the block", the team checked how to combine roll-ups with plasma knowledge and create a new powerful dark matter to scale Ethereum even further!
    
During the research it became obvious that every protocol is special and needs a custom solution in order to gain traction. Therefore the plasma chain will still be an interesting approach for specific scalable zero-knowledge proofs.

### - Improved development, on the testnet
At the moment the leap mainnet is under construction. We concentrate our efforts on the functionality and reliability of the testnet.

The testnet is ready to get some traffic, and all products can be tested, before getting launched on main. It will all fall into place, once the brain/code work is done.

Soon, there will be a return of the mainnet with improved features and a variety of new updates. Maintaining a mainnet demands various resources: effort, money & manpower. All of these are rare and hard to obtain... 
    
### - Hard facts & The next steps

The official mainnet shutdown is expected for the **1st of March 2020**. Until then, the Leap Plasma chain is in operation and working. In full [holacracy fashion](https://leapdao.org/blog/Minimal-Viable-Governance), a governance proposal will be submitted, to enable the next proceedings.

Leap Plasma contracts will be upgraded and unclaimed funds are recovered, in order to distribute them to their respective owners via root-chain. Further deposits are not allowed.

Once the contract upgrades are finalized, the actual shutdown is done by stopping all the Leap Mainnet validator nodes and all the related infrastructure. Funds are withdrawn & returned.

**--> If you have any funds on Leap Mainnet, please make sure to withdraw them. <--**
As our mainnet pause is starting on the 01.03.2020, we advise you to take action before that date.

You can do that very easy over here: https://mainnet.leapdao.org

### - Following our purpose and driving for mass adaption
Whatever is the way we go and which technologies we adopt, the team is motivated to work towards our common purpose:

"Help collectives of people to collaborate on things they care about and achieve sovereignty in a non-violent and fulfilling way."

The aim is to preserve our decentralised & fully-distributed team, get some engines running and help the crypto/ethereum ecosystem achieving mass-adoption.

**Comment by simonweniger** [comment]
@zschavi check the Blogpost here : https://test.leapdao.org/blog/mainnet-shutdown
maybe we need a short summery for the first block ? 
and we need a one or two sentence summery for the link preview

**Comment by zschavi** [comment]
first block:
the leap network is restructuring, we explore new technologies & upgrade the testnet.

summary:
Leap Mainnet on hold, here is why:

**Comment by zschavi** [comment]
https://test.leapdao.org/blog/mainnet-shutdown/

**Comment by simonweniger** [comment]
https://github.com/leapdao/leapdao-website/pull/144 @johannbarbie or @sunify or @troggy  plz. review for merging
